### ISSUE TITLE
[a11y][Violation][2.4.1 Bypass Blocks] Multiple elements with "navigation" role on the Notebook UI do not have unique labels

### ISSUE THREAD

**Issue by nkn2022** [jupyterlab/jupyterlab#17461]
## Description

I am running an accessibility test suite with Jupyter using the **IBM Equal Access Accessibility Checker** tool on Chrome browser. The results look good overall, only a few things are missing to reach a first formal level of compliance. So kindly help look into these. 

Issues reported were violating **WCAG 2.2** requirements. 
Ref: https://www.ibm.com/able/requirements/checker-rule-sets

Listing of violations categorized based on the checkpoints. 

# Category: `2.4.1 Bypass Blocks`
Ref: https://www.ibm.com/able/requirements/requirements/?version=v7_3#2_4_1

****Violation of Rule:** `aria_navigation_label_unique`**

> List of issues with elements that violate this rule with a screenshots
> > **Issue 1** 
> > Multiple elements with "navigation" role do not have unique labels
> > `<div role="navigation" id="menu-panel" class="lm-Widget lm-Panel">`

<img width="480" alt="Image" src="https://github.com/user-attachments/assets/246ebe34-1b15-4f68-b841-0de3a4a1efb6" />

> **What to do**
> For each element with a "nav" role, either set the value of the aria-labelledby attribute to the id value of any visible text that appropriately and uniquely describes that navigation region
> Or, use the aria-label attribute to add an invisible, unique, non-empty label to an element with the "nav" role

## Reproduce

- Install Chrome extension for IBM accessibility checker tool. Ref: https://www.ibm.com/able/toolkit/verify/automated
- Open JupyterLab UI on Chrome browser. Open `notebook` file to edit.
- Run the tool and we can see the above `Violation`s reported.

## Context

- Operating System and version: MacOS Sonoma Version 14.7.3
- Browser and version: Chrome Version 134+
- JupyterLab version: Version 4.3.6

**Comment by danyeaw** [comment]
Hi @nkn2022, thanks for the issue report. Could you please clarify which elements have the navigation role without unique labels?

Querying `$$('[role="navigation"]')` is only returning one array with that role.

**Comment by nkn2022** [comment]
@danyeaw Yes, there is only that one element with `id` in the UI. The report had some duplicates reports but all point to the same element on the UI. I will update the title.

**Comment by nkn2022** [comment]
@danyeaw I double checked. This issue is reported because in this case the notebook UI is embedded inside another UI shell which has its own `nav` elements. When there are multiple navigation element it is expected that all of them have unique `arial-label` property. So adding this prop to this element is required.

**Comment by nkn2022** [comment]
This issue is not reported anymore. Now there is only a single element with `navigation` role displayed on the UI. So no action needed from Jupyter team now. Closing.
